cypress/e2e/crypto/decryption-failure.spec.ts

```diff
@@ -113,10 +135,10 @@ describe("Decryption Failure Bar", () => {
                 })
                 .then(() => {
                     cy.botSendMessage(bot, roomId, "test");
-                    cy.contains(
-                        ".mx_DecryptionFailureBar .mx_DecryptionFailureBar_message_headline",
-                        "Verify this device to access all messages",
-                    );
+                    cy.get("[data-testid='decryption-failure-bar-headline-waiting']").should("exist");
```

Comment:
We should try to use assertions in end to end tests that are close to how a user would experience the app. So rather than using testids here to check the failure bar is displaying the right state, it would be better to assert the correct text is showing (like the old assertion here that you replaced). Since each state uses unique text it should be a simple swap.

Reply:
Right, I was probably considering the possibility of conducting a test in another language, but it will require a general policy which covers the overall UI.

Reply:
@kerryarchibald another suggestion: the fact that E2E tests should emulate how the user is expected to experience suggests me that this test should not only check if a part of strings exists but also check if the all of them are really visible (it is crucial to ensure that they do not overflow, because this is the issue what this PR addresses).